app/views/catalog/_save_search.html.erb

```diff
@@ -8,7 +8,8 @@
         <h4 class="modal-title" id="save-modal-label"><%= t(:'spotlight.saved_search.label') %></h4>
       </div>
       <div class="modal-body">
-        <%=f.text_field :title, label: t(:'spotlight.saved_search.title'), required: true %>
+        <%= f.text_field :title, label: t(:'spotlight.saved_search.title') %>
+        <%= select_tag :id, options_for_select(current_exhibit.searches.map { |s| [s.title, s.id] }), include_blank: true, label: t(:'spotlight.saved_search.id') %>
```

Comment:
Yeah, looks like I broke it updating the parameter name (although, apparently, not in a way the tests catch..). We need `select_tag` to ensure we get `params[:id]` instead of `params[:search][:id]`, but I guess that means we need to be explicit about everything.